Sources/LoopAlgorithm/AutomaticDoseRecommendation.swift

```diff
@@ -9,11 +9,33 @@
 import Foundation
 
 public struct AutomaticDoseRecommendation: Equatable {
-    public var basalAdjustment: TempBasalRecommendation?
+    
+    public enum Direction: String, Codable {
+        case decrease
+        case neutral
+        case increase
+        
+        static func from(correction: InsulinCorrection) -> Self? {
+            switch correction {
+            case .inRange:
+                return .neutral
+            case .aboveRange:
+                return .increase
+            case .entirelyBelowRange:
+                return .decrease
+            case .suspend:
+                return nil
```

Comment:
I would expect .suspend to return `.decrease`; this represents the case when any point in the forecast hits the suspend threshold.

Reply:
Okay that fix is pushed!